wiki/Guides/id.md

```diff
@@ -0,0 +1,52 @@
+# Panduan
+
+Dibawah ini tersedia panduan yang telah dibuat oleh anggota dari komunitas osu!; sebagian besar panduan berasal dari forum osu! dan sudah pindahkan ke wiki osu!.
+                                                                
+## Beatmapping
+
+*Halaman utama: [Beatmapping](/wiki/Beatmapping)*
+
+- [Menambahkan kustom Hitsounds](Adding_Custom_Hitsounds)
+- [Editing Audio](Audio_Editing)
+- [Mengubah nama Artist atau judul](Changing_the_Artist_or_Title)
+- [Mengubah judul dari text](/wiki/Beatmap/Title_Text#changing-title-text)
+- [Informasi Kolaborasi](Collab_Information)
+- [Mengompresi File](Compressing_Files)
+- [Kustom Hitsound Library](Custom_Hitsound_Library)
+- [Kustom Sample Overrides](Custom_Sample_Overrides)
+- [Mengambil lagu dari permainan](Getting_Songs_From_Video_Games)
+- [Mengatur Timing Musik](How_to_Time_Songs)
+- [Teori musik](Music_Theory)
+- [Panduan mapping osu!mania](osu!mania_Mapping_Guide)
+- [Pengaturan Dari Offset pada beat yang tepat](Setting_the_Offset_on_the_Correct_Beat)
+- [Memulai Proyek Beatmap](Starting_a_Beatmap_Project)
+- [Video Dari Youtube](Videos_From_Youtube)
+
+## Modding
+
+*Halaman utama: [Modding](/wiki/Modding)*
+
+- [Mendapatkan Mod pada Map Kamu](Getting_Your_Map_Modded)
+
+## Permainan
+
+- [Panduan Pemula](Beginner's_Tutorial)
+- [Cara Menggunakan Offset Wizard](How_to_Use_the_Offset_Wizard)
+- [Mencari dan Mengunduh Beatmaps](Searching_and_Downloading_Beatmaps)
+- [Pembelian Tablet](Tablet_Purchase)
+- [Tips dan Trick Untuk Mengimprove Skill](Tips_and_Tricks_on_Skill_Improvement)
+
+## Membuat skin
+
+*Halaman utama: [Panduan membuat skin dan utas-utas penting](/wiki/Skinning/Guides_and_important_threads)*
+
+- [Memangkas dengan latar belakang Kompleks](Cropping_with_Complex_Backgrounds)
+- [Memangkas dengan latar belakang Simple](Cropping_with_Simple_Backgrounds)
+- [Membuat Spinner untuk berada di tengah secara tepat](Making_Properly_Centered_Spinners)
+
+## Lainya
```

Comment:
```suggestion
## Lainnya
```

Reply:
Thx u for find my typoo :( i already change it anyway